src/amo/reducers/addonsByAuthors.js

```diff
@@ -1,125 +1,165 @@
 /* @flow */
-import type { AddonType, ExternalAddonType } from 'core/types/addons';
+import deepcopy from 'deepcopy';
+import invariant from 'invariant';
+
 import { createInternalAddon } from 'core/reducers/addons';
+import type { AddonType, ExternalAddonType } from 'core/types/addons';
 
 
 type State = {
-  byAddonSlug: { [string]: AddonType },
+  // TODO: It might be nice to eventually stop storing add-ons in this
+  // reducer at all and rely on the add-ons in the `addons` reducer.
+  // That said, these are partial add-ons returned from the search
```

Comment:
Ah, right. I thought I'd requested in a previous review to reflect this in the type. No worries, I'll file a follow-up issue.

Reply:
This reducer has always been storing incomplete add-ons, but moving to using `loadAddons` would be quite a large patch *and* would waste HTTP requests. In reality we need a generalised `partialAddon` reducer or something. Or maybe even a `isPartialAddon` flag in the `addons` reducer.

Reply:
Hah, looks like @willdurand was ahead of us: #3957